README.md

```diff
@@ -1,15 +1,111 @@
-# Group_G_UNO-Card-Game
-Git Repository for SYSC5709F Project
+# Group G - UNO Express Card Game
 
-Group Information
-Organization: Carleton Unviersity (SYSC 5709F)
+* [Application Overview](#application-overview)
+* [Group Information](#group-information)
+* [File Structure](#file-structure)
+* [How to Compile and Test](#how-to-compile-and-test)
+* [Documentation](#documentation)
 
+## Application Overview
 
-***********************************************************
-Authors:
-1. Mia Wei (GitHub: MiaYWei)
-2. Krithiga Subramaniam Ravichandran (GitHub: krithigasr)
-3. Hongbo Pang (GitHub: HongBo713)
-4. Aditi Bhardwaj (GitHub: aditibhardwaj98)
+UNO Express is a fun and interesting game, traditionally played by a group of 2 to 4 people with a  deck of printed cards. We are proposing to develop this interactive game in C programming  language which supports a single player competing against a computer opponent. Playing games online  with computers could speed up our responses and develop logic skills. 
+
+And during the COVID-19  pandemic, we can use this game to keep ourselves entertained while maintaining our social circle.  What’s more, this game will keep you engaged and help reduce stress. This time you neither need to  fret over saying UNO before you drop your penultimate card nor wait until the deck is shuffled.
+
+By  this project, we intend to develop a UNO game engine that can make choices xadhering  to the rules of the game. 
+This game is similar to the well known UNO game but following modifications are made to deliver this game efficiently as a single player game with a computer opponent.
+
+* UNO Express card game has 56 cards, we removed the 'Reverse' action card as it doesn't make much of a sense in a two player game.So,
```

Comment:
"make much of a sense"? or "make much sense"?

Reply:
Both are right!